lethal-squirrels/Bangazon-Site
# Mp account settings

## REFERENCE

Ticket: #17 

## DESCRIPTION
1. View user Profile
2. Edit Profile
3. Added Profile Controller
4. Links to Add or delete payment options
5. Link to view Order History

## STEPS TO TEST
1. pull down branch `mpAccountSettings`
2. Log in
3. Test functionality
	* clikc on My Profile in the Nav
	* Click on Edit Profile and Save something
	* Payment Type Options should direct you to add or delete user payment options
